samcli/commands/local/cli_common/options.py

```diff
@@ -55,6 +55,36 @@ def template_click_option():
                         help="AWS SAM template file")
 
 
+def service_common_options(f):
+    """
+    Common CLI Options that are shared for service related commands ('start-api' and 'start-lambda')
+
+    Parameters
+    ----------
+    f function
+        Callback passed by Click
+
+    Returns
+    -------
+    function
+        The callback function
+    """
+    service_options = [
+        click.option('--host',
+                     default="127.0.0.1",
+                     help="Local hostname or IP address to bind to (default: '127.0.0.1')"),
+        click.option("--port", "-p",
+                     default=3000,
```

Comment:
`start-lambda` should start in a different port by default. So you can use it with `start-api` without extra configuration. What do you think?

Reply:
Good call. I think just doing a +1 is enough. Not sure if the delta is a big deal. Let me know if you think something greater is better.